lib/double_doc/doc_extractor.rb

```diff
@@ -1,8 +1,8 @@
 module DoubleDoc
   class DocExtractor
     TYPES = {
-      'rb' => /\s*##\s?(.*)$/,
-      'js' => %r{\s*///\s?(.*)$}
+      'rb' => /(^\s*|\s+)##\s?(?<documentation_line>.*)$/,
+      'js' => %r{(^\s*|\s+)///\s?(?<documentation_line>.*)$}
```

Comment:
That's the real question, too many repos to check. You'd want to compare the generated documentation for them. I think your change is the safest.